src/ol/reproj/tile.js

```diff
@@ -0,0 +1,325 @@
+goog.provide('ol.reproj.Tile');
+
+goog.require('goog.asserts');
+goog.require('goog.events');
+goog.require('goog.events.EventType');
+goog.require('goog.math');
+goog.require('goog.object');
+goog.require('ol.Tile');
+goog.require('ol.TileState');
+goog.require('ol.extent');
+goog.require('ol.math');
+goog.require('ol.proj');
+goog.require('ol.reproj');
+goog.require('ol.reproj.Triangulation');
+
+
+
+/**
+ * @classdesc
+ * Class encapsulating single reprojected tile.
+ * See {@link ol.source.TileImage}.
+ *
+ * @constructor
+ * @extends {ol.Tile}
+ * @param {ol.proj.Projection} sourceProj Source projection.
+ * @param {ol.tilegrid.TileGrid} sourceTileGrid Source tile grid.
+ * @param {ol.proj.Projection} targetProj Target projection.
+ * @param {ol.tilegrid.TileGrid} targetTileGrid Target tile grid.
+ * @param {number} z Zoom level.
+ * @param {number} x X.
+ * @param {number} y Y.
+ * @param {number} pixelRatio Pixel ratio.
+ * @param {function(number, number, number, number) : ol.Tile} getTileFunction
+ *     Function returning source tiles (z, x, y, pixelRatio).
+ * @param {number=} opt_errorThreshold Acceptable reprojection error (in px).
+ * @param {boolean=} opt_renderEdges Render reprojection edges.
+ */
+ol.reproj.Tile = function(sourceProj, sourceTileGrid,
```

Comment:
Adding it only to `ol.reproj.Image` would not be very useful -- it would have to be added also to `ol.source.Image` and passed from all the subclasses, but I don't think it's worth the extra code:
`opt_renderEdges` is used for debugging only and it's probably enough to have this in tiled sources.
I also think that `opt_errorThreshold` is kind of "advanced" parameter for fine-tuning and nobody needs it when displaying image sources.
But if you think otherwise, I don't have any problem adding these (or just the `opt_errorThreshold`).